cli/f3.go

```diff
@@ -0,0 +1,168 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/go-f3/manifest"
+)
+
+var (
+	F3Cmd = &cli.Command{
+		Name:  "f3",
+		Usage: "Manages Filecoin Fast Finality (F3) interactions",
+		Subcommands: []*cli.Command{
+			{
+				Name:    "list-participants",
+				Aliases: []string{"lp"},
+				Usage:   "Lists the miners that currently participate in F3 via this node.",
+				Action: func(cctx *cli.Context) error {
+					api, closer, err := GetFullNodeAPIV1(cctx)
+					if err != nil {
+						return err
+					}
+					defer closer()
+
+					participants, err := api.F3ListParticipants(cctx.Context)
+					if err != nil {
+						return cli.Exit(fmt.Errorf("listing participants: %w", err), 1)
```

Comment:
We exit with exit code 1 when returning an error anyways so this shouldn't be necessary:

https://github.com/filecoin-project/lotus/blob/fda61d33b30973512afb26aff2dce007db4ec313/cli/helper.go#L48-L59